### ISSUE TITLE
Add continous ping mode to traceroute

### ISSUE THREAD

**Issue by Crazy-KSK** [winsiderss/systeminformer#2026]
When doing a traceroute on an ip in the network tab it would be very useful to have the ability to continuously ping all ips in the route.

Thanks!

btw I noticed there being four column labels named time instead of min, max, average, I assume that's just WIP?

**Comment by gvanem** [comment]
> btw I noticed there being four column labels named time instead of min, max, average, 

No. it sends max. 4 (`DEFAULT_MAXIMUM_PINGS`) ICMP Echo requests to each hop. 
In this case **[1]** the router (in Belgium) at hop 9:

![Tracert](https://github.com/winsiderss/systeminformer/assets/945271/d1021add-732e-42fa-8dc8-40ab5b0554d1)


there is no reply on the 1st and 2nd `ICMP_ECHO_REQUEST`. Similar for router at hop 11 (in Beijing).

**[1]**: tracing the path to Voice Of Korea's web-site (`www.vok.rep.kp`). Good luck with getting through,
